src/Libraries/Microsoft.Extensions.AI.Abstractions/ChatCompletion/IChatClient.cs

```diff
@@ -9,6 +9,20 @@
 namespace Microsoft.Extensions.AI;
 
 /// <summary>Represents a chat completion client.</summary>
+/// <remarks>
+/// <para>
+/// Unless otherwise specified, all members of <see cref="IChatClient"/> are thread-safe for concurrent use.
+/// It is expected that all implementations of <see cref="IChatClient"/> support being used by multiple requests concurrently.
+/// </para>
+/// <para>
+/// However, implementations of <see cref="IChatClient"/> may mutate the arguments supplied to <see cref="CompleteAsync"/> and
```

Comment:
"May" seems a bit weak given that mutating the list is the standard mechanism for updating the chat history.
```suggestion
/// However, implementations of <see cref="IChatClient"/> do mutate the arguments supplied to <see cref="CompleteAsync"/> and
```

Reply:
```suggestion
/// However, implementations of <see cref="IChatClient"/> are permitted to mutate the arguments supplied to <see cref="CompleteAsync"/> and
```